From 9ac03500773a1cc69b198ce1c43770a7fc66da98 Mon Sep 17 00:00:00 2001
From: Andrew Robbins <contact@andrewrobbins.info>
Date: Tue, 5 Sep 2017 23:57:34 -0400
Subject: [PATCH] Preserve unicode source files thru `git clean -df`

UnicodeData.txt and Blocks.txt should be preserved after a
`git clean -df` to avoid needless refetching of these resources.
---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index dedd2b5..841e748 100644
--- a/.gitignore
+++ b/.gitignore
@@ -2,3 +2,5 @@ build/
 dist/
 tmp/
 /src/*.sfd~
+resources/Blocks.txt
+resources/UnicodeData.txt
-- 
1.9.1

